build.gradle

```diff
@@ -1,7 +1,7 @@
 plugins {
   id "com.github.hierynomus.license" version "0.14.0"
   id "com.xebialabs.xl.docker" version "1.1.0"
-  id "nebula.release" version "6.0.0"
+  id "nebula.release" version "11.0.0"
```

Comment:
With version > 6.0, if the latest commit isn't tagged, builds will fail.  Is this something we're ok with?  Here's a minimal build.gradle that provides the same functionality as ≤6.0 nebula.release, but is an up-to-date active package.  I don't know the best approach.  What do you think?

Reply:
https://github.com/xebialabs-community/xlr-qualys-plugin/blob/master/build.gradle

Reply:
for minimal disruption, i for now switched to 8.0.0 of nebula that works.